.github/workflows/iroha2-dev-pr-label.yml

```diff
@@ -0,0 +1,42 @@
+name: I2::Dev::Label
+
+on:
+  pull_request_target:
+    branches: [iroha2-dev]
+    paths:
+      - 'docs/source/references/schema.json'
+      - 'docs/source/references/api_spec.md'
+      - 'docs/source/references/config.md'
+
+jobs:
+  api-changes:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: tj-actions/changed-files@v29.0.3
```

Comment:
The same thing and discussion that related to [PR #2680](https://github.com/hyperledger/iroha/pull/2680#discussion_r964673683)